imports/deprecated/mixins/mixins.Shareable.js

```diff
@@ -45,17 +44,16 @@ const Shareable = {
     const item = this.getShareableEntry(nextProps);
     if (!item) return;
 
-    if (
-      !nextProps.currentSermon &&
-      !nextProps.studyEntry
-    ) {
+    if (!nextProps.currentSermon && !nextProps.studyEntry) {
       this.setShareAction(item);
     } else if (
       (nextProps.currentSermon && nextProps.series) ||
-      (nextProps.studyEntry ** nextProps.study)
+      (nextProps.studyEntry && nextProps.study)
```

Comment:
`(nextProps.studyEntry ** nextProps.study)` was returning `NaN` aka `false`. Because of this devos were never having the `setShareAction` called.